babel.config.js

```diff
@@ -4,19 +4,42 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-// This file is used by jest
-module.exports = {
-  plugins: ['@babel/plugin-proposal-class-properties'],
-  presets: [
-    [
-      '@babel/preset-env',
-      {
-        targets: {
-          node: 'current',
+const pkg = require('./package.json');
```

Comment:
It was hard to correctly configure the rollup plugins and babel-jest to use different config files and they are all happier using this one with a the check for test env

Reply:
Could still do separate files if you do something like
```ts
return isTest ? require('./babel.config.jest.js') : require('./babel.config.rollup.js');
```
This would keep the two completely separate, which might make things easier to untangle in the future.

Another option with similar abstraction but just one file could be
```ts
return isTest ? getJestConfig() : getRollupConfig();
```

At the very least, you should add comments to explain what each config is used for.